jsoc/admins.md

```diff
@@ -3,11 +3,10 @@
 The JSoC (encompasses GSoC, GSoD, etc.) program admins have a few responsibilities including, but not limited too, reviewing contributor applications for all programs, creating and submitting the organization application for each program, helping ensure the programs run smoothly, recruiting mentors, making final contributor selections and more. 
 
 The current JSoC admins are: 
-- [Avik Sengupta](https://twitter.com/aviksengupta) - Julia Project Lead at NumFOCUS
-- [Chris Rackauckas](https://twitter.com/ChrisRackauckas)
-    - (Currently leads [SciML under NumFocus](https://sciml.ai/dev/#google_summer_of_code) as well)
-- [Logan Kilpatrick](https://twitter.com/OfficialLoganK) - Julia Language Community Manager
-- [Viral Shah](https://twitter.com/Viral_B_Shah) - Julia co-creator and CEO of JuliaHub
+- [Avik Sengupta](https://filebox.ece.vt.edu/~aviksg/) - Julia Project Lead at NumFOCUS
```

Comment:
@aviks is that you? 🤔

Reply:
Probabky best to keep the original link 😆

Reply:
```suggestion
- [Avik Sengupta](https://twitter.com/aviksengupta) - Julia Project Lead at NumFOCUS
```
Oops, thanks!